app/views/posts/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>This is a post</h1>
```

Comment:
Please uses `recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page.